pkg/internal/checkup/configmap/configmap_test.go

```diff
@@ -36,8 +36,9 @@ func TestNew(t *testing.T) {
 	name := "my-cm"
 	ownerName := "my-pod"
 	ownerUID := "1234567890"
+	data := map[string]string{}
```

Comment:
Could you please add a couple of keys and values to the data?

Reply:
DONE